src/intl/messages.js

```diff
@@ -134,10 +134,11 @@ export const messages: { [messageId: string]: MessageType } = {
     message: 'Thin Provision',
     description: 'Select list option for qcow / thin provisioned format disk images',
   },
-  diskEditorResizeLabel: 'Extend Size By (GiB)',
+  diskEditorResizeLabel: 'Increase Size By (GiB)',
```

Comment:
Why the wording change?

Reply:
https://github.com/oVirt/ovirt-web-ui/pull/1104#pullrequestreview-283618258